coindesk.go

```diff
@@ -9,8 +9,15 @@ import (
 	"os"
 	"strconv"
 	"strings"
+	"time"
 )
 
+type Price struct {
+	date time.Time
```

Comment:
Rename `date` and `price` to `Date` and `Price`, so that user can access it. Change wherever you mentioned it and update example.

Read the Travis CI details.